internal/certificate/testing.go

```diff
@@ -0,0 +1,82 @@
+package certificate
+
+import (
+	"context"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hetznercloud/hcloud-go/hcloud"
+	"github.com/terraform-providers/terraform-provider-hcloud/internal/testsupport"
+	"github.com/terraform-providers/terraform-provider-hcloud/internal/testtemplate"
+)
+
+func init() {
+	resource.AddTestSweepers(ResourceType, &resource.Sweeper{
+		Name:         ResourceType,
+		Dependencies: []string{},
+		F:            Sweep,
+	})
+}
+
+// Sweep removes all certificates from the backend.
```

Comment:
```suggestion
// Sweep removes all certificates from the Hetzner Cloud API.
```

"Backend" is a to a generic term. In that case, it could also be the terraform backend, or whatever.
(Have a look at the other occurrences too).